### ISSUE TITLE
CC0 licensing ramifications

### ISSUE THREAD

**Issue by slyrz** [rust-fuzz/targets#10]
Writing fuzzer targets should be straight forward, releasing them under CC0 is nice. But what if you want to provide corpus data from an existing MIT/BSD/Apache project? Say you want to fuzz *servo/rust-url*, you could build a URL corpus from the test data of Servo, Chromium, WebKit ... but can you actually push such a corpus to this repository under the current licensing?

**Comment by Manishearth** [comment]
Generally tests are cc0. But not always.

We can add specific licenses to folders

**Comment by frewsxcv** [comment]
Yep, what @Manishearth said. We would need to somehow isolate each licensed corpus and indicate their license somewhere.
